src/pages/utilities/UnrecognizedUtilityTabs/IgnoredFilesTab.tsx

```diff
@@ -96,6 +96,7 @@ function IgnoredFilesTab() {
   const filesQuery = useFilesInfiniteQuery(
     {
       pageSize: 50,
+      include: ['Ignored'],
```

Comment:
This is definitely a server bug because it was working before da3d touched it.

Reply:
include_only should either add this to include silently or throw an error telling you that you are trying to filter something that will never be in the dataset in the first place